docs/source/release/v3.9.0/imaging.rst

```diff
@@ -8,17 +8,18 @@ Imaging Changes
 Tomographic reconstruction graphical user interface
 ###################################################
 
-- Running local reconstructions is now possible, only a single reconstruction process can be run at any one time
+- Running local reconstructions is now possible, only a single reconstruction process can be run at any one time.
+- You can now use Auto-find button in the ROI tab to calculate the Center of Rotation for a sample using TomoPy. The calculated Center of Rotation is automatically loaded back into the GUI. Both the Center of Rotation relative to the crop and the full image are printed. Remote submission for finding the Center of Rotation can be done via the Custom Command tool.
 
 Bug Fixes
 ---------
-- The external interpreter and scripts paths are no longer ignored and are now correctly appended when running a local reconstruction
-- Local reconstruction processes are now also updated in the reconstruction jobs list
-- The reconstruction jobs list command line is now Read Only
-- Clicking Cancel now cancels the running reconstruction process
-- The reconstruction scripts will now work with TomoPy v1.x.x, however the results have yet to be tested
-- Selecting the Center of Rotation, Area for Normalsation and Region of Interest will now always follow the exact position of the mouse
+- The external interpreter and scripts paths are no longer ignored and are now correctly appended when running a local reconstruction.
+- Local reconstruction processes are now also updated in the reconstruction jobs list.
+- The reconstruction jobs list command line is now Read Only.
+- Clicking Cancel now cancels the running reconstruction process.
+- The reconstruction scripts will now run with TomoPy v1.x.x, however the output is not tested with versions newer than 0.10.x.
```

Comment:
Is there any way to query TomoPy for its version an provide a warning for users at runtime?